src/shared/components/Workspace/Queries/queries-board.less

```diff
@@ -0,0 +1,22 @@
+.queries-board {
+  // 117 = 40 (header) + 42 (project banner) + 35 (footer)
+  height: calc(100vh - 117px);
```

Comment:
cannot we automate that with Flexbox or CSS Grid on the whole container of header + body + footer?

Reply:
There's probably a way to do that but it will require some learning on my part

Reply:
I'll experiment with flexbox and flex-grow to get this effect

Reply:
I'll leave this to another patch because those changes will reorder the other pages that we don't have consistent solutions for yet, especially the org lists and such.